Missionframework/modules/05_garrison/fnc/fn_garrison_spawnSectorInfantry.sqf

```diff
@@ -0,0 +1,78 @@
+/*
+    KPLIB_fnc_garrison_spawnSectorInfantry
+
+    File: fn_garrison_spawnSectorInfantry.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-10-20
+    Last Update: 2018-10-23
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Spawns an infantrie squad with given amount of soldiers of given side at given sector.
```

Comment:
Shouldn't it be `infantry` instead of `infantrie` ? :P

Reply:
... ^^